Ziffity/AddCustomer/Profiles/ProfileInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Ziffity\AddCustomer\Profiles;
+
+interface ProfileInterface
+{
+    /**
+     * @param string $file
+     * @return array
+     */
+    public function getData($file);
```

Comment:
@msakila532 method signature missing. pls check all the methods have signatures.